hstream-admin-server/src/HStream/Admin/Server/Connector.hs

```diff
@@ -0,0 +1,199 @@
+import           Data.Aeson                       (FromJSON, ToJSON)
+import qualified Data.ByteString.Char8            as C
+import           Data.Int                         (Int64)
+import           Data.List                        (find)
+import qualified Data.Map.Strict                  as Map
+import           Data.Maybe                       (fromMaybe)
+import qualified Data.Text                        as T
+import qualified Database.ClickHouseDriver.Client as ClickHouse
+import qualified Database.ClickHouseDriver.Types  as ClickHouse
+import qualified Database.MySQL.Base              as MySQL
+import           GHC.Generics                     (Generic)
+import           RIO                              (async, forM_, forever)
+import           Servant                          (Capture, Delete, Get, JSON,
+                                                   PlainText, Post, ReqBody,
+                                                   type (:>), (:<|>) (..))
+import           Servant.Server                   (Handler, Server)
+import qualified Z.Data.CBytes                    as CB
+import qualified Z.Data.CBytes                    as ZDC
+import qualified Z.Data.Text                      as ZT
+import           Z.IO.Time                        (SystemTime (..),
+                                                   getSystemTime')
+import qualified ZooKeeper.Types                  as ZK
+
+import           HStream.Connector.ClickHouse     (clickHouseSinkConnector)
+import qualified HStream.Connector.HStore         as HCH
+import           HStream.Connector.MySQL          (mysqlSinkConnector)
+import           HStream.Processing.Connector     (SinkConnector (..),
+                                                   SourceConnectorWithoutCkp (..),
+                                                   subscribeToStream)
+import           HStream.Processing.Processor     (getTaskName,
+                                                   taskBuilderWithName)
+import           HStream.Processing.Type          (Offset (..), SinkRecord (..),
+                                                   SourceRecord (..))
+import qualified HStream.SQL.AST                  as AST
+import qualified HStream.SQL.Codegen              as HSC
+import           HStream.SQL.Exception            (SomeSQLException)
+import qualified HStream.Server.Exception         as HSE
+import           HStream.Server.Handler           (catchZkException,
+                                                   runTaskWrapper)
+import qualified HStream.Server.Persistence       as HSP
+import qualified HStream.Store                    as HS
+import           HStream.Utils.Converter          (cbytesToText, textToCBytes)
+
+data ConnectorBO = ConnectorBO
+  {
+    id          :: Maybe T.Text,
+    status      :: Maybe Int,
+    createdTime :: Maybe Int64,
+    sql         :: T.Text
+  } deriving (Eq, Show, Generic)
+instance ToJSON ConnectorBO
+instance FromJSON ConnectorBO
```

Comment:
Format this code and consider replacing ConnectorBO with a more readable name.

Reply:
format how?

Reply:
comma leading?

Reply:
I thought ConnectorBO will be fine? 
https://topic.alibabacloud.com/a/brief-description-of-object-types-common-in-java-do-bo-dto-vo-ao-po_1_27_30001659.html